testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -260,13 +260,16 @@ def test_check_coordinate_limits_max_noconversion(self):
 
 
 class TestGROWriterLarge(TestCase, tempdir.TempDir):
-    def setUp(self):
-        self.tmpdir = tempdir.TempDir()
-        self.large_universe = mda.Universe(GRO_large)
 
-    def tearDown(self):
-        del self.tmpdir
-        del self.large_universe
+    @classmethod
```

Comment:
http://nose.readthedocs.io/en/latest/writing_tests.html?highlight=setupclass#test-classes

and for usage in a class that descends from `unittest.TestCase`:
http://stackoverflow.com/questions/7020781/unittest-setupclass-not-working